metadata-ingestion/schedule_docs/cron.md

```diff
@@ -0,0 +1,28 @@
+# Using Cron
+
+To follow this guide you need to use [DataHub CLI](../../docs/cli.md).
+
+Assume you have a receipe file `/home/ubuntu/example.yml` on your machine
```

Comment:
name the file to be more descriptive? e.g.`/home/ubuntu/datahub_ingest/mysql_to_datahub.yml`